compiler/parser/parser.peg

```diff
@@ -1036,7 +1036,7 @@ ComplexType
 
 TemplateLiteral
   = v:TemplateLiteralParts {
-      if ISNULL(v) {
+      if ARRAY_ISLEN(v, 0) {
```

Comment:
My vote's for `if (ARRAY_LEN(v) == 0) {` but this is OK.

Reply:
I'll try this.